### DISCUSSION TITLE
Documentation regarding backups

### DISCUSSION THREAD

**Discussion by vlvrd** [coder/coder#13271] [General] [answered]
After reviewing the docs, I noticed that the [Reference Architectures](https://coder.com/docs/v2/latest/admin/architectures#database-backups) section doesn't answer me a couple of questions:
- Are backups for older server versions useful for newer versions?
- Which information is backed up? Can workspaces info be restored?
- Is there a guide with proper Coder backup guidelines? Is it just a Postgresql DB backup or should I consider something else?

Is it possible to clarify these in the documentation?

**Comment by kylecarbs** [comment, marked as answer]
We'll update the docs!

> Are backups for older server versions useful for newer versions?

Yes! We've never had a breaking migration. Even the oldest versions of Coder can be upgraded!

> Which information is backed up? Can workspaces info be restored?

Only metadata is preserved in the database. Any infrastructure you provide is not backed up (e.g., PVCs, compute, etc.).

> Is there a guide with proper Coder backup guidelines? Is it just a Postgresql DB backup or should I consider something else?

There's no guide right now. Backing up the PostgreSQL database would suffice!
